django_prbac/tests/test_models.py

```diff
@@ -71,10 +71,16 @@ def test_has_permission_immediate_params(self):
         self.assertFalse(subrole.instantiate({}).has_privilege(superrole1.instantiate(dict(one='baz'))))
 
     def test_unsaved_role_does_not_have_permission(self):
+        import django
         role1 = Role()
         role2 = arbitrary.role()
-        self.assertFalse(role1.has_privilege(role2))
-        self.assertFalse(role2.has_privilege(role1))
+        if django.VERSION >= (4, 0):
+            with self.assertRaises(ValueError):
+                self.assertFalse(role1.has_privilege(role2))
+                self.assertFalse(role2.has_privilege(role1))
```

Comment:
Which of the above two assertions raises `ValueError`? If it's the first then the second would never run, and therefore could be omitted. Also I assume `assertFalse` is never called. Would this change make sense?
```suggestion
            with self.assertRaises(ValueError):
                role1.has_privilege(role2)
```
Having written that, I'm wondering if `has_privilege` should be updated to handle the error and return false?

Reply:
Yeah this was sloppy on my part. I updated this to have `get_privileges` handle the potential ValueError raised. 9c0f27706e9dd3ede3e6aeb4e216c9e446156a16